### PR TITLE
Fix clicks on the download notification

### PR DESCRIPTION
**Pull request by MatkovIvan**
* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you add unit tests?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

App doesn't resume when the user clicks on the download notification.

## Related PRs or issues

#1613
[AB#91962](https://dev.azure.com/msmobilecenter/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/91962)

### CODE REVIEW

File: sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeUtils.java [hunk 1]
```diff
@@ -47,15 +55,62 @@ class DistributeUtils {
...
     static int getNotificationId() {
         return Distribute.class.getName().hashCode();
     }
 
+    static void postDownloadNotification(@NonNull Context context, String message, Intent intent) {
+        NotificationManager notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+        Notification.Builder builder;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+
+            /* Create or update notification channel (mandatory on Android 8 target). */
```
Comment (DmitriyKirakosyan) [inline sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeUtils.java:78]: on Android 8 and higher?

File: sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeUtils.java [hunk 2]
```diff
@@ -47,15 +55,62 @@ class DistributeUtils {
...
+    }
+
+    static void cancelDownloadNotification(@NonNull Context context) {
+        NotificationManager notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+        notificationManager.cancel(DistributeUtils.getNotificationId());
+    }
+
+    @NonNull
+    @SuppressWarnings({"deprecation", "RedundantSuppression"})
+    private static Notification.Builder getOldNotificationBuilder(@NonNull Context context) {
```
Comment (DmitriyKirakosyan) [inline sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeUtils.java:110]: This "one-line" function is used in one place only. Do we need this function?

Comment (MatkovIvan) [reply sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeUtils.java:110]: Yes, to suppress warning

File: sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeUtils.java (hunk 1 above)
Comment (MatkovIvan) [reply sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeUtils.java:78]: Sure, that what is comment and code about

**Review by AnatolyPristensky — APPROVED** [review]

**Pull request merged**